imsv-docs-astro/src/content/docs/guides/regions.mdoc

```diff
@@ -2,7 +2,18 @@
 title: Supported regions
 ---
 
-A crypto holder must reside in one of the countries listed below to be able to order and use Immersve cards.
+A crypto holder must reside in one of the supported regions to be able to
+order and use Immersve cards.
+
+The availability of certain regions may vary based on the partner card program:
+a region may be supported in principle but might not be accessible to a given
+partner if their card program(s) is not set up in that region.
```

Comment:
`program(s)` has such a bureaucracy smell to it 😂 do we actually use this pattern in other guides?

Reply:
bureaucracy smell 😄

Reply:
I reworded, see if that is better

Reply:
much better, thanks ❤️